server/src/test/scala/com/ossum/riddl/lsp/server/RiddlLSPTextDocumentSpec.scala

```diff
@@ -170,14 +172,28 @@ class RiddlLSPTextDocumentSpec
       }
     }
 
-    /* TODO: Finish these tests immediately
     "successfully close everythingOneError.riddl" in new OpenOneErrorFileSpec {
       val closeNotification: DidCloseTextDocumentParams =
         new DidCloseTextDocumentParams()
       closeNotification.setTextDocument(textDocumentIdentifier)
       service.didClose(closeNotification)
+
+      val params = new CompletionParams()
+      params.setTextDocument(textDocumentIdentifier)
+      val position = new Position()
+      position.setLine(1)
+      position.setCharacter(1)
+      params.setPosition(position)
+
+      val resultF: CompletableFuture[
+        messages.Either[util.List[CompletionItem], CompletionList]
+      ] = service.completion(params)
+
+      // yields failure because file is closed so doc is not accessible
+      resultF.asScala.failed.futureValue mustBe a[UnavailableResourceException]
```

Comment:
Is this the only testable thing?

Reply:
Yes, there is really no other way to make sure the file is closed except by returning an exception